comms/lighthouse/src/peers/peersService.ts

```diff
@@ -143,4 +144,38 @@ export class PeersService implements IPeersService {
         .slice(0, 100)
     )
   }
+
+  sendChangeToIsland(
```

Comment:
`sendIslandChange` and `sendChangeToIsland` are very similar in name. Could we make them a little bit more different. Maybe `informIslandAboutUpdate` and `sendNewIslandToPeer`?